Quantum-Control-Applications/AMO/Use Case 1 - Saffman Lab - Atom Sorting/Array_sorting.py

```diff
@@ -0,0 +1,497 @@
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm import SimulationConfig
+from qm import LoopbackInterface
+from Config_array_sorting import *
+import matplotlib.pyplot as plt
+
+
+################################
+# Open quantum machine manager #
+################################
+qmm = QuantumMachinesManager()

```

Comment:
add host= and ip= and add them to the config (same as SC config, check there)

Reply:
Done